doc/devices.rst

```diff
@@ -26,10 +26,16 @@ Supported operations and observables
 
     ~pennylane.BasisState
     ~pennylane.CNOT
+    ~pennylane.CRot
+    ~pennylane.CRX
+    ~pennylane.CRY
+    ~pennylane.CRZ
```

Comment:
Good spot!

Reply:
In the long run, maybe we shouldn't even have this list of supported ops! Every op should be supported :zap: (and saves us having to maintain this)

Reply:
Sure --- do you think this entire list can be pulled? Or should we opt for that later?